### DISCUSSION TITLE
Centrar toplevel en pantalla

### DISCUSSION THREAD

**Discussion by lildinti** [alejandroautalan/pygubu-designer#129] [Q&A] [answered]
Como ubicar toplevel en centro de pantalla?
Cheers 
Claudio

**Comment by alejandroautalan** [comment, marked as answer]
Hello Claudio, sorry for the late response (didn´t get the notification)

I made an example that answer your question.
https://github.com/alejandroautalan/pygubu-designer/tree/master/examples/toplevel_centered

Let me know.

PD: You can open any discussion theme, here on the pygubu-designer discussions.

**Comment by lildinti** [comment]
Hi Alejandro,
No worries. I know you're a busy man!
Thanks for your help. FinishinG  new example of phonebook using JSON as backend.
Cheers, 
Claudio
